### ISSUE TITLE
mrDeadcell (st4rgard3n)

### ISSUE THREAD

**Issue by St4rgarden** [raid-guild/RIPs#103]
### Championee

MrDeadce11

### Champion

st4rgard3n

## Summary of work to date

MrDeadce11 has done:
- some documentation for the Raider's Handbook
- a significant amount of smart contract development work (testing) on the Soul Bound Token Staking Cohort (RiteOfMoloch)
- is working on the Metastreet raid currently; earning his first Raid Bread again doing smart contract development work

### Applicant source

 - [X] Cohort process (`25,000 RAID` stake)
 - [ ] Non-cohort (`50,000 RAID` stake)

### Link to staking transaction (to `0x45b49f854eebadbdb59ede30fe5b37ba56a0f26c`)

https://blockscout.com/xdai/mainnet/tx/0xbb33b45fdf5f593b6e503f31c8efebaaa6acea9b1b493243a00d89d57c94b1df

## Anything else you'd like to add?

MrDeadce11 has proven himself to be a competent fledgling smart contract developer.  He may still require some support during his next few Raids by more senior or experienced developers - but overall he's very solid and determined.  He makes up for his lack of experience with sheer grit and tenacity.

## Success criteria

Comment within this issue, before moving to membership, the realization of the
following success criteria:

1. Apprentice period of 60 days. (Success)
2. Successfully join at least 1 raid or RIP. This includes raids or RIPs
   completed as well as raids or RIPs still in progress (provided the
   apprentice is in good standing on that project). (In progress already at Milestone #3) Payment receipts: https://blockscout.com/xdai/mainnet/tx/0xff7cc3e82de2063d59f76a5fbeeeb509137c06b4f666faa37c79baed053f66b8
https://blockscout.com/xdai/mainnet/tx/0x5e5bdc67838db83278644329d46d69ea000fde93b5cf860b610cadeeba394aa3
3. Attend at least 2 core Guild meetings, currently includes Roundtable, Raider
   Roundup, or Token Talk. (He's always there we would need bear mace to get rid of him)
4. Demonstrate shared values with the Guild (at the discretion of the DAO).

**Comment by plor** [comment]
Eligible for membership proposal on: Saturday, November 12, 2022

**Comment by MrDeadCe11** [comment]
The metastreet -seaport integration raid with st4rgard3n on which I wrote the test suit for a contract that St4r engineered was a fantastic experience all around and I got some great feedback from st4r as well as the client which has really helped me develop as a developer.  Hopefully we'll be seeing more work from them before too long.

Currently I'm working with bitbeckers, plor, jip, tufnel, and tae on a lenster - gitcoin integration that I'm super excited about.

I couldn't have asked for a better champion than st4rgard3n.  Thank you sir for everything you've done and continue to do to help me along the path.  Mad props to you my friend.    Let's  FUCKING GOOOOOOOOO!
